### ISSUE TITLE
Align "Explore" toolkit with new design

### ISSUE THREAD

**Issue by microbit-matt-hillsdon** [microbit-foundation/python-editor-next#400]
Blocked on #376.

![image](https://user-images.githubusercontent.com/44397098/147255320-a7ec57c0-57e6-4cd7-897a-de7b623712d6.png)

- New colours
- New larger icons at the top level but not at the second level
- CodeEmbed changes quite a bit. Maybe keep wider drag handle.
- Code colours, let's start with just the background not the text changes.
- Insert code button icon should be from remix. Maybe it looks odd? Just an arrow?

For the toolkit header area we have different content from the design. 

Let's do this:

![image](https://user-images.githubusercontent.com/44397098/147255988-4bdaf19a-0d62-46d2-95a0-c77be1f6b3bd.png)

but with the subtitle from the CMS, which is shorter.

Then we need to add in the introduction field from the CMS as we have currently and experiment with what looks good.

After we've done that we can consider the compact form shown on scroll.

Not in scope:
- "Remove code", highlighting of code after insert, highlighting of code box

This issue might benefit from being split up or tackled as multiple PRs.

**Comment by microbit-matt-hillsdon** [comment]
This has been merged and we've agreed to leave it at this point.
